os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -1705,11 +1706,10 @@ static struct iahwc_backend *iahwc_backend_create(
   iahwc_module_t *iahwc_module;
   iahwc_device_t *iahwc_device;
 
-  /* const char* device = "/dev/dri/renderD128"; */
-  const char *device = "/dev/dri/card0";
+  const char* device = "/dev/dri/renderD128";
```

Comment:
Can you explain in commit message the problem and how is it fixed.

Reply:
Okay.